domain.go

```diff
@@ -312,6 +312,7 @@ type DomainGraphicListener struct {
 }
 
 type DomainGraphic struct {
+	XMLName       xml.Name                `xml:"graphics"`
```

Comment:
I am using it as separate document when changing graphics config via domain.UpdateDeviceFlags(dev_xml, libvirt.DOMAIN_DEVICE_MODIFY_CURRENT) and without this patch it marshals into <DomainGraphic> tags.